### PR TITLE
Add RMRK Multi resource NFT to dev3

### PR DESCRIPTION
**Pull request by stevyhacker**
Adds required files to support the creation of RMRK Multi resource NFT collections through dev3 protocol.

- RMRKMultiResource.sol is a flattened RMRKMultiResourceImpl contract from the official RMRK EVM repo.
- info.md you can find a short description.
- manifest.json is a platform-specific metadata file.
- artifact.json is the contract's ABI

### CODE REVIEW

File: contracts/rmrk/multiresourcenft/manifest.json [hunk 1]
```diff
@@ -0,0 +1,1114 @@
...
+        {
+          "name": "Resource ID",
+          "description": "Resource ID",
+          "recommendedTypes": []
+        }
+      ],
+      "returnDecorators": [
+        {
+          "name": "Resource id",
+          "description": "Returns a resource ID",
```
Comment (steven2308) [inline contracts/rmrk/multiresourcenft/manifest.json]: 0 if the proposed resource is not overriding anything.

File: contracts/rmrk/multiresourcenft/manifest.json [hunk 2]
```diff
@@ -0,0 +1,1114 @@
...
+          "description": "Returns a number of max tokens in the collection",
+          "recommendedTypes": []
+        }
+      ],
+      "emittableEvents": []
+    },
+    {
+      "signature": "mint(address,uint256)",
+      "name": "Mint",
+      "description": "Mints a new token to an address",
```
Comment (steven2308) [inline contracts/rmrk/multiresourcenft/manifest.json]: Mints new tokens to an address

File: contracts/rmrk/multiresourcenft/manifest.json [hunk 3]
```diff
@@ -0,0 +1,1114 @@
...
+          "description": "Returns price of the minting of a new token",
+          "recommendedTypes": []
+        }
+      ],
+      "emittableEvents": []
+    },
+    {
+      "signature": "rejectAllResources(uint256)",
+      "name": "Reject all resources",
+      "description": "Rejects all resources for a given token id",
```
Comment (steven2308) [inline contracts/rmrk/multiresourcenft/manifest.json]: Rejects all the pending resources for a given token id.

File: contracts/rmrk/multiresourcenft/manifest.json [hunk 4]
```diff
@@ -0,0 +1,1114 @@
...
+          "recommendedTypes": []
+        }
+      ],
+      "returnDecorators": [],
+      "emittableEvents": []
+    },
+    {
+      "signature": "rejectResource(uint256,uint256)",
+      "name": "Reject a resource",
+      "description": "Rejects a resource for a given token id and resource id",
```
Comment (steven2308) [inline contracts/rmrk/multiresourcenft/manifest.json]: Rejects a pending resource for a given token id and index (of pending resources array)

File: contracts/rmrk/multiresourcenft/manifest.json [hunk 5]
```diff
@@ -0,0 +1,1114 @@
...
+          "recommendedTypes": []
+        }
+      ],
+      "returnDecorators": [],
+      "emittableEvents": []
+    },
+    {
+      "signature": "setPriority(uint256,uint16[])",
+      "name": "Set priority",
+      "description": "Sets priorities for the given token id",
```
Comment (steven2308) [inline contracts/rmrk/multiresourcenft/manifest.json]: Sets priorities for the given token id. They are assigned to active resources on the corresponding positions.

File: contracts/rmrk/multiresourcenft/manifest.json [hunk 6]
```diff
@@ -0,0 +1,1114 @@
...
+          "recommendedTypes": []
+        }
+      ],
+      "returnDecorators": [],
+      "emittableEvents": []
+    },
+    {
+      "signature": "setTokenEnumeratedResource(uint64,bool)",
+      "name": "Set Token Enumerated Resource",
+      "description": "Sets token enumerated resource state",
```
Comment (steven2308) [inline contracts/rmrk/multiresourcenft/manifest.json]: Sets token enumerated resource state. If this is active for a resource, the tokenId is appended when getting TokenURI for it.

File: contracts/rmrk/multiresourcenft/manifest.json [hunk 7]
```diff
@@ -0,0 +1,1114 @@
...
+          "description": "Token URI calculated for the given token ID. Usually it is constructed by appending the token ID value to the base URI defined at the collection construction time.",
+          "recommendedTypes": []
+        }
+      ],
+      "emittableEvents": []
+    },
+    {
+      "signature": "tokenURIAtIndex(uint256,uint256)",
+      "name": "Token URI at Index",
+      "description": "Returns token URI for given token ID and index. Token URI describes the actual digital content connected to the given token. This can for example be a link to the image file if the nft itself is an image or digital art.",
```
Comment (steven2308) [inline contracts/rmrk/multiresourcenft/manifest.json]: Returns token URI for given token ID and **resource** index. [...]

**Review by steven2308 — COMMENTED** [review]
Mostly minor clarifications. This looks awesome!

**Comment by stevyhacker** [comment]
I have included @steven2308 suggestions and updated the contract version. It's ready for review now @filipdujmusic

**Pull request merged**